homework1_portfolio.html

```diff
@@ -0,0 +1,54 @@
+<html>
+
+<head>
+	<title>Portfolio Page</title>
+	<link rel="stylesheet" type="text/css" href="homework1_portfolio_styles.css">
+</head>
+
+<body>
+	<div><hr color="#000000" size="4" width="1175px" align= "center"></div>
+			
+			<nav>
+				<a href="homework1_aboutme.html">About Me</a> | <i>Portfolio</i>
+			</nav>
+			
+			<h1>Joell Shepperson</h1>
+
+			<div><hr color="#000000" size="1" width="1025px" align= "center"></div>
+
+
+	<h2>Experience</h2>
+				
+		<p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.</p>
+		
+	<h2>Portfolio</h2>
+		
+		<h4><i>Web Developer,</i> Relaxr 2015 to present</h4>
+			<p>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</p><img src="images/relaxr_deliverable.png" alt="Relaxr" height="250" width="250">
+
+		<div class="secondblock"><h4><i>Web Developer</i>, Startup Matchmaker 2015 to present</h4></div>
```

Comment:
Nice use of nested tags here.  Next time, definitely be sure to start each tag on a new (indented) line.  It will make reading your HTML much easier - especially as you start working on more complicated pages.